contracts/test/modules/Offers/V1/Offers.t.sol

```diff
@@ -110,6 +110,11 @@ contract OffersV1Test is DSTest {
         offers.createOffer(address(token), 0, address(0), 1 ether, 1000);
     }
 
+    function testFail_CannotCreateOfferWithInvalidFindersFeeBps() public {
+        vm.prank(address(maker));
+        offers.createOffer(address(token), 0, address(0), 1 ether, 10001);
```

Comment:
`10001` bps (basis points) is just one higher than what the `require()` looks for:

https://github.com/ourzora/v3/blob/60b2875a80c6cdfb13bab838b7da00c283b5b346/contracts/modules/Offers/V1/OffersV1.sol#L159